### PR TITLE
Run/Stop buttons visibility

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- Enable run button only if there is at least one computational service

#### Bonus:
- Double clicking on a FolderViewer File item opens/downloads file

startStop

## Related issue/s
closes https://github.com/ITISFoundation/osparc-issues/issues/400

### CODE REVIEW

**Comment by KZzizzle** [comment]
Very cool idea! I think that I might rather have a grayed-out run button with a tooltip that says "nothing to run" or something like that as opposed to getting rid of it completely... just in case it could be confusing for someone who is used to seeing the run button? Of course you can also say why that's not preferable =)

File: services/web/client/source/class/osparc/desktop/StartStopButtons.js [hunk 1]
```diff
@@ -65,7 +63,8 @@ qx.Class.define("osparc.desktop.StartStopButtons", {
     setRunning: function(running) {
       const startButtons = [this.__startButton, this.__startSelectionButton.getChildControl("button"), this.__startAllButton];
       startButtons.forEach(startButton => startButton.setFetching(running));
-      this.__stopButton.setEnabled(running);
+
+      this.__stopButton.setVisibility(running ? "visible" : "excluded");
```
Comment (KZzizzle) [inline services/web/client/source/class/osparc/desktop/StartStopButtons.js:67]: for me this is not necessary to have

**Review by KZzizzle — APPROVED** [review]

File: services/web/client/source/class/osparc/desktop/StartStopButtons.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/StartStopButtons.js:67]: Reverted 👍

**Comment by odeimaiz** [comment]
> Very cool idea! I think that I might rather have a grayed-out run button with a tooltip that says "nothing to run" or something like that as opposed to getting rid of it completely... just in case it could be confusing for someone who is used to seeing the run button? Of course you can also say why that's not preferable =)

Done, without the tooltip.

**Pull request merged**